src/components/notification/notification.component.ts

```diff
@@ -223,17 +222,17 @@ export class NotificationComponent implements OnInit, OnDestroy {
   }
 
   private instance = () => {
-    this.diff = (new Date().getTime() - this.start) - (this.count * this.speed);
+    var now = new Date().getTime();
```

Comment:
`now` is never updated, please use `const`.

Reply:
Oky 👍